### PR TITLE
 Beta Part 2 Pull Projects

### PR DESCRIPTION
**Pull request by noahmalmed**
This PR finishes off the beta review Code.

- Add Beta Projects to main projects call
- Add UI for Beta Projects section
- Add redux state for beta projects

# Review Checklist

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

File: src/components/ProjectDisciplines.js [hunk 1]
```diff
@@ -150,10 +153,11 @@ export class ProjectDisciplines extends React.Component {
       </FontedText>
 
     const disciplineInProjectList = (discipline) => {
-      const {user, hasPreviewProjects, hasRecentProjects} = this.props
+      const {user, hasPreviewProjects, hasRecentProjects, hasBetaProjects} = this.props
```
[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/components/ProjectDisciplines.js]: addressed

[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/components/ProjectDisciplines.js]: addressed

**Comment by wgranger** [comment]
What is the reason for the change in `WebViewScreen`?

**Comment by noahmalmed** [comment]
Before this change when the webview would load it would just show a blank white screen. I just gave the webview a spinner so the user will know that something is happening while the screen loads

**Comment by wgranger** [comment]
Thanks for the explanation on the webview. I think the loading spinner is a nice touch and the code here looks pretty clean. Merging.

**Pull request merged**